### PR TITLE
Add spacesuit avatar asset

### LINKED ISSUE #1663: New avatar assets
### User Story

As a user, I want to new avatar assets to show off and express myself digitally.

### Description

*Type*:  New implementation

### Definition of Done

- [ ] Create an .svg asset and place it in `app/assets/v2/images/avatar/<category>`
- [ ] Add the SVG asset as a `.txt` Django template in `app/avatar/templates/<category>/<item|index>.txt` with variable fill.
- [ ] Add your asset to [utils.py](https://github.com/gitcoinco/web/blob/master/app/avatar/utils.py#L43-L97)
- [ ] Provide a screenshot of the new asset from the avatar builder AND the profile details page on this Github issue.

### Additional Information

Feel free to submit any assets you would like to see on Gitcoin.  If the asset is accepted, your asset will become a part of the live avatar builder!  All other assets will be escalated to the core team for review and potential modification.

See the [avatar item request shortlist](https://github.com/gitcoinco/web/issues/1550)

**COMMUNITY MEMBERS**
Feel free to add reactions/upvote your favorite new avatar items/assets!

### PR DESCRIPTION
**Pull request by ajayns**
##### Description
Adds spacesuit clothing asset to avatar builder. Currently I've only worked on one color, but if the design looks good, I'll add the other colors and we could move forward.

screenshot from 2018-08-21 13-57-11

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)A

##### Refers/Fixes
#1663

### CODE REVIEW

**Comment by owocki** [comment]
@mbeacom i commented on the another PR... but is there anything beyond adding the files to the filesystem that bounty hunters need to do to add their items? im not able to get this to work on my local

@ajayns sorry, i'm dense. what about this garment makes it a space suit?

**Comment by ajayns** [comment]
Haha, I was trying to figure out a way to make it look like a spacesuit without a helmet and found the task quite difficult. After a few iterations I assumed this would make sense, but apparently it doesn't. I'll rethink my design and come up with something better and which conveys the idea of a spacesuit. 

Also, if you had any ideas of how it should look like (which I could use as inspiration) that'd be great! :)

**Comment by thelostone-mc** [comment]
Same [comment](https://github.com/gitcoinco/web/pull/2047#issuecomment-415332111) as the other one

- compress + wire up + screenshot

**Comment by ajayns** [comment]
Yes I was thinking of ideas, got quite caught up with work before. Now I've been experimenting a bit and here goes:
screenshot from 2018-08-25 23-40-31

**Comment by ajayns** [comment]
screenshot from 2018-08-26 15-10-44

I was parallelly working on a helmet as well, please give your feedback!

**Comment by owocki** [comment]
i like it; we'll need to figure out how this goes into the avatar builder though!

**Comment by thelostone-mc** [comment]
You'll have to add it [here](https://github.com/gitcoinco/web/blob/master/app/avatar/utils.py#L71) 
It looks like this @owocki @ajayns 

screen shot 2018-08-28 at 12 33 04 am

**Comment by ajayns** [comment]
@thelostone-mc Oh that step wasn't mentioned in the bounty issue. All it said was copy the assets and update the .txt file.

**Comment by thelostone-mc** [comment]
@ajayns ah sorry about that ! I guess we forgot to type it in
It's a one word addition! Mind adding it in ?

**Comment by ajayns** [comment]
@thelostone-mc Yes yes, will do. I mentioned it here cause then you could add it to the original issue and it won't confuse the other bounty hunters :)

**Comment by thelostone-mc** [comment]
@ajayns way ahead of you :P

**Comment by ajayns** [comment]
@PixelantDesign @owocki If this design looks good to you guys I can move on to adding the all the versions and other steps for integration of these new items. 

I'm parallelly troubleshooting issues with my local setup, I'm unable to provide screenshots for the builder. Although using its specs it's supposed to look like the sample image I sent which includes the face and helmet.

**Comment by ajayns** [comment]
@thelostone-mc Updated with all the integrations! 

I'll open a separate PR for the helmet once @PixelantDesign approves it, else we can work on iterations.

**Comment by ajayns** [comment]
I tried out a couple of variations, these looked good to me. Opinions guys?

1:
screenshot from 2018-08-30 09-36-45

2:
screenshot from 2018-08-30 09-36-28

**Comment by ajayns** [comment]
@thelostone-mc Hey, I still haven't gotten the build issue fixed, so I'd appreciate if you could test out and post screenshots thanks :)

@PixelantDesign Please review the helmet design so that I can move forward with a next PR

**Comment by ajayns** [comment]
@thelostone-mc The helmet is a separate accessory so I thought I'd make a separate PR for that once @PixelantDesign approves

**Comment by PixelantDesign** [comment]
first one is looking good!

**Comment by thelostone-mc** [comment]
@ajayns might updating it within this PR ? ^_^
Makes my life a lil easier in terms of reviewing

**Comment by ajayns** [comment]
If the spacesuit looks good, let's merge it @thelostone-mc @PixelantDesign 

I'll be opening another PR for the helmet

**Comment by ajayns** [comment]
@thelostone-mc Sure! Whatever's convenient by you. I'll push the changes

File: app/avatar/utils.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def get_avatar_context():
             'options': (['Glasses-0'], ['Glasses-1'], ['Glasses-2'], ['Glasses-3'], ['Glasses-4'], [
                 'HatShort-backwardscap'
             ], ['HatShort-ballcap'], ['HatShort-headphones'], ['HatShort-shortbeanie'], ['HatShort-tallbeanie'],
-                        ['Earring-0'], ['Earring-1'], ['EarringBack-2', 'Earring-2'], ['Earring-3'], ['Earring-4'])
+                        ['Earring-0'], ['Earring-1'], ['EarringBack-2', 'Earring-2'], ['Earring-3'], ['Earring-4'], ['Helmet-space'])
```
[the remark this answers is not in the document]
Comment (ajayns) [reply app/avatar/utils.py]: My linter config isn't set up, so if this works fine can you fix run it through the linter before merge?

**Comment by thelostone-mc** [comment]
@ajayns this is what it looks like
x


It looks weird if the user picks an hairstyle 🤔 

without it , the grey helmet and colored spacesuit looks odd 😓 
screen shot 2018-09-03 at 8 33 06 pm


cc @PixelantDesign

**Comment by ajayns** [comment]
> It looks weird if the user picks a hairstyle 

Nothing can be done about that, we'll have to rely on the user to make wise decisions. Alternative would be to make a huge helmet, which I don't think would look good or be a good idea.

> without it, the grey helmet and colored spacesuit looks odd

Currently, accessories don't have an option for color picking, so the solution for this could adding the helmet with the clothing asset itself but that'd also mean users have no option to wear just the spacesuit and not the helmet.

**Comment by owocki** [comment]
maybe we remove the hair (or we have a little piece of text that recommends they remove the hair) when they selected the space helmet (or other accessories that are better without hair)

**Comment by PixelantDesign** [comment]
If we wont allow any  bangs or hair around the face, we should crop the face of the helmet in tighter so that we cant see edge of face or ears.

To solve for hair issue, if user selects helmet, remove hair option and notify the user that they can choose either **hair** or the **helmet**.

Thoughts?

**Comment by thelostone-mc** [comment]
I'm up for putting the suit and helmet as clothing and leaving to the user to make the sensible decision of leaving the avatar bald ! 
Else might require adding a lot of conditional logic which at this point might be an overkill 


> If we wont allow any bangs or hair around the face, we should crop the face of the helmet in tighter so that we cant see edge of face or ears.

I agree with this

**Comment by ajayns** [comment]
screenshot from 2018-09-05 10-25-44

Variation @PixelantDesign suggested

**Comment by thelostone-mc** [comment]
@ajayns after discussing with @PixelantDesign .
She felt that we should wait this one out -> until we enable coloring on accessories which might not happen anytime in the future

Closing this one out for now !
Thanks for helping out with this one ^_^

**Comment by ajayns** [comment]
Sure! Although maybe you could use just the spacesuit clothing asset, in the case helmet wouldn't work out now.

**Comment by thelostone-mc** [comment]
Yeah i was open to that but defer to @PixelantDesign who felt we should roll out both together

**Comment by ajayns** [comment]
Okay sure! Thanks for all your feedback on this. :)

**Pull request closed without merging**